src/mixins/nr-affiliation-mixin.ts

```diff
@@ -141,27 +141,29 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
     let businessRequest = {} as BusinessRequest
     const nrNumber = nr.nrNum
     if (!this.isFirm(nr)) {
-      name = FilingTypes.INCORPORATION_APPLICATION
-      legalType = this.entityTypeToCorpType(nr.entity_type_cd)
-      businessRequest = {
-        filing: {
-          business: { legalType },
-          header: { accountId, name },
-          incorporationApplication: {
-            nameRequest: { legalType, nrNumber }
+      if (nr.request_action_cd === 'AML') {
```

Comment:
This looks correct now - thanks.

If you're ever in here again -- perish the thought! -- use `NrRequestActionCodes.AMALGAMATE` instead of "AML". (This isn't worth a new commit on its own.)